build/tools.sh

```diff
@@ -38,11 +38,25 @@ function install_git-lfs() {
   rm -rf /tmp/lfs.tar.gz "/tmp/git-lfs-${GIT_LFS_VERSION}"
 }
 
+function configure_docker_group_id() {
```

Comment:
This should go around [here](https://github.com/myoung34/docker-github-actions-runner/blob/ddaa37c0295a92eb119932f5d708e54129d5d03b/build/install_base.sh#L57) instead, no need to make it a function, just do `groupadd -g "$(docker_group_id)" docker || :`

Reply:
You can add it after https://github.com/myoung34/docker-github-actions-runner/blob/ddaa37c0295a92eb119932f5d708e54129d5d03b/build/install_base.sh#L52 just before install_tools_apt and install_tools so that it exists before those run, just add a comment above such as `# The docker group needs to run before installers` or similar